test/integration/helpers/Npm.cs

```diff
@@ -0,0 +1,73 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Appium.Net.Integration.Tests.helpers
+{
+    internal class Npm
+    {
+
+        public static string GetNpmPrefixPath()
+        {
+
+            string npmPath = "npm";
+
+            if (Platform.CurrentPlatform.IsPlatformType(PlatformType.Windows))
+            {
+                npmPath = GetNpmExecutablePath();
+            }
+
+            string npmPrefixPath = RunCommand(npmPath, "-g root");
+
+            return npmPrefixPath.Trim();
+        }
+
+        private static string RunCommand(string command, string arguments)
+        {
+            int timeoutMilliseconds = 30000;
+            try
+            {
+                Process process = new Process
+                {
+                    StartInfo = new ProcessStartInfo
+                    {
+                        FileName = command,
+                        Arguments = arguments,
+                        RedirectStandardOutput = true,
+                        UseShellExecute = false,
+                        CreateNoWindow = true,
+                    }
+                };
+
+                process.Start();
+                string output = process.StandardOutput.ReadToEnd();
+                process.WaitForExit(timeoutMilliseconds);
```

Comment:
does this call also verify the exit code?

Reply:
it does not, you see a point in adding this verification?

Reply:
yes. Any non-zero response would mean a failure